DMX.Core.Api/Services/Orchestrations/LabWorkflows/LabWorkflowOrchestrationService.cs

```diff
@@ -31,6 +31,20 @@ public partial class LabWorkflowOrchestrationService : ILabWorkflowOrchestration
             this.loggingBroker = loggingBroker;
         }
 
+        public ValueTask<LabWorkflow> SubmitLabWorkflowAsync(LabWorkflow labWorkflow) =>
+        TryCatch(async () =>
+        {
+            ValidateLabWorkflowOnSubmit(labWorkflow);
+
+            await this.labWorkflowService.AddLabWorkflowAsync(labWorkflow);
+
+            labWorkflow.Commands.ForEach(async command =>

```

Comment:
Yeah, it's causing some issues with EntityFramework: 

`"A second operation was started on this context instance before a previous operation completed. This is usually caused by different threads concurrently using the same instance of DbContext. For more information on how to avoid threading issues with DbContext, see https://go.microsoft.com/fwlink/?linkid=2097913."` 

Looking into it

Reply:
Resolved with latest commit